query-engine/query-engine-wasm/example/prisma/schema.prisma

```diff
@@ -0,0 +1,13 @@
+datasource db {
```

Comment:
We remove the `example` directory on the main branch, we should not add it again, it looks like your PR is in a weird state.

Reply:
yeah,there are merge artifacts. I will just remove whatever looks weird.